content/articles/deploying-a-serverless-django-restapi-with-zappa-on-aws/index.md

```diff
@@ -0,0 +1,440 @@
+
+![Deployment](/engineering-education/deploying-a-serverless-jango-restapi-with-zappa-on-aws/meme.jpg)
+
+[Image source](https://www.google.com/url?sa=i&url=https%3A%2F%2Fmakeameme.org%2Fmeme%2Fbrace-yourself-deploy&psig=AOvVaw30-YKcrTN4IWmGoZeWsnAT&ust=1637270554714000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCPiBt6GqoPQCFQAAAAAdAAAAABAJ)
+
+
+### Setting up AWS
+
+At this point, it is supposed that you have created your AWS account and have access to an AWS IAM user for the Zappa utility. Zappa utility handles tasks such as automatically creating an s3 bucket for deployment, creating lambda execution IAM roles, API gateways for web traffic work, etc. on your behalf.
+
+Briefly, let us set up AWS on your console to properly utilize Zappa.
+
+S*tep 1*: Log in to your AWS console and go to the IAM section
+
+*Step 2*: On the left side of the screen click on *users*
+
+*Step 3*: Click on *add users*. For this project, we will name our 'user', *'serverless'* and for access type, choose '*Programmatic access*'. This uses the API to interact with AWS.
+
+*Step 4*: Move on the permission and select *'Attach existing policies directly*. We will be accessing two policies, the first is '*IAMFullAccess*" This creates users for lambda to be executed. The second is *'PowerUserAccess'* for creating API gateway and S3 bucket. Search for these policies and click on their boxes.
+
+*Step 5*: Skip Add tags, move to Review and create a user.
+
+*Successfully created!* We have also been supplied with an access key and secret access key. Download the *.csv file* which contains these keys as these pieces of information will be lost when the page closes
+
+Briefly, let'sSet up AWS on your console in other to properly utilize Zappa. 
+
+### Deploying with Zappa
+
+The first step is to install Zappa in our virtual environment:
+
+```python
+pip install zappa
+```
+
+Next, initialize Zappa, 
+
+```python
+zapa init
+```
+
+```python
+███████╗ █████╗ ██████╗ ██████╗  █████╗
+╚══███╔╝██╔══██╗██╔══██╗██╔══██╗██╔══██╗
+  ███╔╝ ███████║██████╔╝██████╔╝███████║
+ ███╔╝  ██╔══██║██╔═══╝ ██╔═══╝ ██╔══██║
+███████╗██║  ██║██║     ██║     ██║  ██║
+╚══════╝╚═╝  ╚═╝╚═╝     ╚═╝     ╚═╝  ╚═╝
+
+Welcome to Zappa!
+
+Zappa is a system for running server-less Python web applications on AWS Lambda and AWS API Gateway.
+This `init` command will help you create and configure your new Zappa deployment.
+Let's get started!
+
+Your Zappa configuration can support multiple production stages, like 'dev', 'staging', and 'production'.
+What do you want to call this environment (default 'dev'): dev
+```
+
+Flowing from this instruction, select an environment name. In this project, we used the default name, '*dev'.*
+
+```python
+It looks like this is a Django application!
+What is the module path to your projects's Django settings?
+We discovered: bookstore.settings
+Would you like to deploy this application ←[1mglobally←[0m? (default 'n') [y/n/(p)rimary]: n

```

Comment:
`←[1mglobally←[0m? (default 'n'`

I'm not sure if this is copy-pasted correctly. Please verify

Reply:
Yes it is